commit f56437f2c613f5b2bc8df638aa9da876571771a7
Author: Frans Pop <fjp@debian.org>
Date:   Wed Aug 12 19:35:50 2009 +0200

    Update licence and copyright information
    
    Clarify version of GPL (version 2 or later).
    Update periods of activity for various maintainers both in
    the perl script and in the debian/copyright file.

diff --git a/debian/changelog b/debian/changelog
index f6324ad..56d7f63 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,8 +1,11 @@
 debmirror (2009xxxx) UNRELEASED; urgency=low
 
   * Set more accurate versioned build dependency on debhelper.
+  * Clarify version of GPL (version 2 or later).
+  * Update periods of activity for various maintainers of the script both
+    in the perl script and in the debian/copyright file.
 
- -- Frans Pop <fjp@debian.org>  Tue, 11 Aug 2009 14:38:27 +0200
+ -- Frans Pop <fjp@debian.org>  Wed, 12 Aug 2009 19:33:54 +0200
 
 debmirror (20090807) unstable; urgency=low
 
diff --git a/debian/copyright b/debian/copyright
index a8ef4cd..383c0de 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -1,10 +1,20 @@
 This package was put together by Joerg Wendland <joergland@debian.org>
 on Thu Oct 25 17:12:13 CEST 2001
 
-The upstream author was Joey Hess <joeyh@debian.org>, the script
-itself was maintained by Joerg Wendland <joergland@debian.org> and
-later by Goswin von Brederlow <brederlo@informatik.uni-tuebingen.de>
-for some time, and is now maintained by Frans Pop <fjp@debian.org>.
+The original author of the script is Joey Hess <joeyh@debian.org>.
+Later the script itself was maintained for some time by Joerg Wendland
+<joergland@debian.org> and by Goswin von Brederlow
+<brederlo@informatik.uni-tuebingen.de>.
+It is currently being maintained by Frans Pop <fjp@debian.org>.
 
-License: The debmirror script is licensed under the terms of the GNU
-         General Public License (see /usr/share/common-licenses/GPL).
+Copyright:
+    Copyright 2001      Joey Hess <joeyh@debian.org>
+    Copyright 2001-2002 Joerg Wendland <joergland@debian.org>
+    Copyright 2003-2007 Goswin von Brederlow <brederlo@informatik.uni-tuebingen.de>
+    Copyright 2009      Frans Pop <fjp@debian.org>
+
+License:
+    The debmirror script is licensed under the terms of the GNU
+    General Public License, either version 2 of licence (see
+    /usr/share/common-licenses/GPL-2) or (at your option) any later
+    version.
diff --git a/debmirror b/debmirror
index 433fd03..5b4f156 100755
--- a/debmirror
+++ b/debmirror
@@ -1625,8 +1625,11 @@ sub debug {
 =head1 COPYRIGHT
 
 This program is copyright 2001 by Joey Hess <joeyh@debian.org>, under
-the terms of the GNU GPL, copyright 2003 by Goswin von Brederlow
-<brederlo@informatik.uni-tuebingen.de>.
+the terms of the GNU GPL (either version 2 of the licence or, at your
+option, any later version), copyright 2001-2002 by Joerg Wendland
+<joergland@debian.org>, copyright 2003-2007 by Goswin von Brederlow
+<brederlo@informatik.uni-tuebingen.de> and copyright 2009 by
+Frans Pop <fjp@debian.org>.
 
 The author disclaims any responsibility for any mangling of your system,
 unexpected bandwidth usage bills, meltdown of the Debian mirror network, 
@@ -1634,9 +1637,13 @@ etc, that this script may cause. See NO WARRANTY section of GPL.
 
 =head1 AUTHOR
 
-Current: Goswin von Brederlow <brederlo@informatik.uni-tuebingen.de>
-Previous authors: Joey Hess <joeyh@debian.org>
-                  Joerg Wendland <joergland@debian.org>
+Current maintainer: Frans Pop <fjp@debian.org>
+
+Previous authors:
+
+   Joey Hess <joeyh@debian.org> (original author)
+   Joerg Wendland <joergland@debian.org>
+   Goswin von Brederlow <brederlo@informatik.uni-tuebingen.de>
 
 =head1 MOTTO
 
